network-upgrades/mainnet-upgrades/merge.md

```diff
@@ -0,0 +1,38 @@
+## Merge Upgrade Specification
```

Comment:
```suggestion
## Serenity Upgrade Specification
```
I think you misspelled Serenity.

Reply:
tfw you try and go for the name-agnostic spec...!

Reply:
oof saw this before i saw your discord post haha

Reply:
Serenity is (if anything) the meta-name!

Serenity is the effort to make ethereum more secure, sustainable, and scalable, but the Merge only handles the first two